packaging/release

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env node
+
+/* eslint-disable import/no-extraneous-dependencies, no-console */
+const git = require('simple-git/promise');
+const util = require('util');
+const exec = util.promisify(require('child_process').exec);
+const { argv: { branch, platform, token } } = require('yargs');
+const checkVersion = require('./check-version');
+const updateRemoteVersion = require('./update-remote');
+
+const release = async () => {
+  const track = branch === 'dev' ? 'alpha' : 'beta';
+
+  // Get all tags, sort in desc order, and get latest from current track
+  const gitTags = await git().tags();
```

Comment:
you could pass a sort function to the tags method.

Reply:
When I do that, I get an old tag that starts with 'v' that always comes up first.